JS101/DOM-manipulation.md

```diff
@@ -1,16 +1,258 @@
-Here's where we introduce putting JS and HTML together.  DOM Manipulation with Javascript.
+One of the most unique and useful abilities of javascript is it's ability to manipulate the DOM. But what *is* the DOM, and how do we go about changing it? Let's jump right in...
 
-It could be that this needs to be split into multiple lessons.. that'll depend on how it plays out once we start writing it.
+### Points to Ponder
 
-we should 100% definitely stick to PLAIN JS.  Not jQuery.  might even be worth mentioning that this is the type of stuff that many people rely on jQuery for.
+1. Do you understand what the DOM is in relation to a webpage?
+2. Do you understand the difference between a "node" and an "element"?
+3. Do you understand how to target nodes with "selectors"?
+4. Do you know the basic methods for finding/adding/removing and altering DOM nodes?
+5. Likewise, do you know the difference between a "nodelist" and an "array of nodes"?
+6. Do you understand how "events" and "listeners" work?
 
-concepts to be covered:
-1. querySelector (querySelectorAll)
-2. classList.add/remove/toggle etc.
-3. innerHtml/textContent
-4. events/eventListeners
-5. appendChild etc.
+### DOM - Document Object Model
 
-this would be a _great_ place to include and recommend JS30.. most of those lessons are just simple DOM-manipulation projects with vanilla JS.
+When working with HTML and JS, you'll often here mention of "the DOM". What the DOM is, on a high level, is a tree-like representation of the contents of a webpage or "document". A tree of "nodes" with different relationsships depending on how they're arranged in the HTML document.
 
-should do some simple example project.  Something simple that requires some user interaction like a tip calculator or something (put in the cost and the tip percentage and have it display the amount onscreen)
\ No newline at end of file
+```
```

Comment:
nitpick... but can you go add a JavaScript tag to all the code blocks like this:
```javascript 

doing so enables syntax highlighting